pyproject.toml

```diff
@@ -125,6 +127,8 @@ module = [
     "tlv8.*",
     "pytest.*",
     "click_aliases.*",
+    "smartcard.*",
+    "asn1crypto.*",
```

Comment:
Ideally I’d like to have typing stubs for new dependencies, but I can also add them in a separate PR.

Reply:
Creating stubs for `asn1crypto` would be pretty painful.

I think we should just migrate to the cryptography.x509 package, in the issue I opened there they gave an example for how to use a custom signer.

Reply:
That would be even better.  But I’m also fine with sticking with `asn1crypto` for the moment and replacing it later.

Reply:
Ok, i'll leave it that way for now. It's experimental anyway.